actions/views/rhs.ts

```diff
@@ -251,6 +281,46 @@ export function showPinnedPosts(channelId?: string) {
     };
 }
 
+export function showChannelFiles(channelId?: string) {
```

Comment:
Is it feasible to make `channelId` required?

Reply:
This was using the same approach already there for other like the `showPinnedPosts`, but yes, it can be changed.

Reply:
Done